diff --git a/node_modules/mocha-chrome/lib/client.js b/node_modules/mocha-chrome/lib/client.js
index 7629126..cd3d163 100644
--- a/node_modules/mocha-chrome/lib/client.js
+++ b/node_modules/mocha-chrome/lib/client.js
@@ -9,7 +9,7 @@ module.exports = async function connectClient(instance, log, options) {
     return fs.readFileSync(filePath, 'utf-8');
   }
 
-  const client = await CDP({ port: instance.port });
+  const client = await CDP({ port: instance.port, host: '127.0.0.1' });
   const { DOM, DOMStorage, Console, Network, Page, Runtime } = client;
   const mochaOptions = `window.mochaOptions = ${JSON.stringify(options.mocha)}`;
 
